### ISSUE TITLE
`fused_weight_gradient_mlp_cuda` module not found. gradient accumulation fusion with weight gradient computation disabled.

### ISSUE THREAD

**Issue by ofishhaha** [Sense-X/UniFormer#24]
I met this problem after installing mmdet2.11 in object_detection and mmcv1.3.14. It seems something wrong of the installation and multi-gpu training. Do u have any idea whats wrong with it?

Thank u for ur efforts.

**Comment by ofishhaha** [comment]
It seems is an apex installation warning. by the way, I'm using cuda9.2 and torch1.7. Would u mind also share ur installation environment?  thank u

**Comment by Andy1621** [comment]
Actually, you do not need to install Apex, because I never use it to train detection models. And using FP16 will drop the performance slightly.

I'm using cuda10.2+torch1.9, but I think cuda10.1+torch1.8 is OK.

Moreover, could you please share the detailed problem you meet? I will help you check it in the MMDetection issue.

**Comment by ofishhaha** [comment]
Thank u for the replay. By the way what’s the version of ur mmcv ？
